internal/pkg/redis/client_test.go

```diff
@@ -408,7 +409,7 @@ func TestClient_Subscribe(t *testing.T) {
 					Messages: make(chan types.MessageEnvelope),
 				},
 				{
-					Topic:    "UnitTestTopic2",
+					Topic:    "UnitTestTopic2/+/test/#",
```

Comment:
I don't think this is sufficient to ensure our topic conversions are working as expected. See other request for new unit tests.